src/Agda/Compiler/Scala/PrintScala3.hs

```diff
@@ -12,13 +12,12 @@ import Agda.Compiler.Scala.ScalaExpr ( ScalaName, ScalaExpr(..), SeVar(..))
 printScala3 :: ScalaExpr -> String
 printScala3 def = case def of
   (SePackage pNames defs) ->
-    (printPackageAndObject pNames)
+    printPackageAndObject pNames
       <> bracket (map printScala3 defs)
       <> blankLine -- EOF
   (SeSum adtName adtCases) ->
-    (printSealedTrait adtName)
-    <> defsSeparator
-    <> combineLinesWithIndent indent (map (printCaseObject adtName) adtCases)
+    printEnum adtName
+    <> bracketWithIndent (map printEnumCase adtCases) 2
```

Comment:
This change modifies output Scala3 files, can you commit those as well:
```sh
% cabal run -- agda2scala --compile --no-main --scala-dialect=Scala3 --out-dir=scala3/src/main/scala ./examples/adts.agda

% git status
Changes not staged for commit:
modified:   scala3/src/main/scala/examples/adts.scala
```

I want to be able to see changes in output Scala files when browsing git history (even though they seems redundant as they are generated in CI.)